--- gxine-0.5.905/gxine.desktop.in.orig	2009-12-19 17:10:03.000000000 +0100
+++ gxine-0.5.905/gxine.desktop.in	2011-02-19 20:42:05.602975068 +0100
@@ -5,14 +5,16 @@
 GenericName[ca]=Reproductor de peŀlícules
 GenericName[de]=Video-Player
 GenericName[es]=Reproductor de películas
+GenericName[pl]=Odtwarzacz filmów
 Comment=Watch videos or TV and listen to audio in a variety of formats
 Comment[ca]=Reproduïu vídeos i cançons
 Comment[es]=Reproduce vídeos y canciones
+Comment[pl]=Odwarzanie filmów i piosenek, oglądanie telewizji cyfrowej
 Exec=gxine %U
 X-MultipleArgs=true
 Icon=gxine
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=AudioVideo;GTK;Player;TV;
+Categories=AudioVideo;GTK;Player;
 MimeType=x-content/video-dvd;x-content/audio-cdda;x-content/video-vcd;x-content/video-svcd;
\ Brak znaku nowej linii na końcu pliku
--- gxine-0.5.905/mime.extra.orig	2009-12-19 17:10:04.000000000 +0100
+++ gxine-0.5.905/mime.extra	2011-02-20 07:23:13.558973392 +0100
@@ -1,3 +0,0 @@
-x-content/video-dvd
-x-content/video-svcd
-x-content/video-vcd
