--- gxine-0.5.910/configure.ac.orig	2024-04-16 22:28:59.967243570 +0200
+++ gxine-0.5.910/configure.ac	2024-04-17 06:24:41.395954723 +0200
@@ -641,7 +641,10 @@ AM_CONDITIONAL(HAVE_OPTIPNG, [test "x$OP
 AM_CONDITIONAL(HAVE_ADVPNG, [test "x$ADVPNG" != x])
 
 
-GXINE_PLUGINDIR="$libdir/gxine"
+AC_ARG_WITH([plugindir],
+  AS_HELP_STRING([--with-plugindir=path],[use path to install browser plugin]),
+  [GXINE_PLUGINDIR=$withval], [GXINE_PLUGINDIR="$libdir/gxine"]
+)
 eval GXINE_PLUGINPATH=`eval echo "$GXINE_PLUGINDIR"`
 AC_DEFINE_UNQUOTED(GXINE_PLUGINDIR,"$GXINE_PLUGINPATH",[where to install the browser plugin])
 AC_SUBST(GXINE_PLUGINPATH)
